lib/sealField.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const iron = require('iron');
+const nodeify = require('./nodeify');
+
+/**
+ * Seal a field
+ * @method sealField
+ * @param  {Object}         model               The model to seal a property of
+ * @param  {Object}         config
+ * @param  {Mixed}          config.field        The field to store the sealed value in
```

Comment:
Each instance where it's being used, it's being passed a `String`. 

https://github.com/screwdriver-cd/models/pull/162/files#diff-326dff258ee0c2713dca9ab6a5d701d3R32
https://github.com/screwdriver-cd/models/pull/162/files#diff-db62f6a8d7d20845993eb9a470a318dfR67
https://github.com/screwdriver-cd/models/pull/162/files#diff-398ed3b014436a5204583323ea29320bR48
https://github.com/screwdriver-cd/models/pull/162/files#diff-1fb87bb8462436f368b6060fd2cfb374R51